### PR TITLE
Removed SWF / Flash library since it is long dead

### PR DESCRIPTION
**Pull request by Lonzak**
This is a first version where I did remove the swf part of the uploader.
It needs code review since my knowledge is limited in this area.

### CODE REVIEW

**Comment by jiakuan** [comment]
Thanks for the pull request. I will find a chance to review over the weekend.

File: gwt-uploader/src/main/java/org/wisepersist/gwt/uploader/client/Uploader.java [hunk 1]
```diff
@@ -297,51 +226,27 @@ public void setHttpHeaders(JSONObject httpHeaders) {
...
-   *                          the Ajax approach and instead utilize the SWFUpload/Flash technique
-   *                          only.
-   * @return A reference to this {@link Uploader} instance for convenient method chaining.
-   */
-  public Uploader setAjaxUploadEnabled(boolean ajaxUploadEnabled) {
-    this.ajaxUploadEnabled = ajaxUploadEnabled;
     return this;
   }
 
+  private boolean ajaxUploadEnabled = true;
```
Comment (jiakuan) [inline gwt-uploader/src/main/java/org/wisepersist/gwt/uploader/client/Uploader.java]: If we only have Ajax upload, do we still need this variable at all?

**Comment by jiakuan** [comment]
Have you got a chance to test the functionality after Flash related stuff have been removed?

File: gwt-uploader/src/main/java/org/wisepersist/gwt/uploader/client/Uploader.java (hunk 1 above)
Comment (Lonzak) [reply gwt-uploader/src/main/java/org/wisepersist/gwt/uploader/client/Uploader.java]: Yes your are right - removed it.

**Comment by Lonzak** [comment]
I did update to gwt 2.10.0 however there are certain problems:
The gradle plugin  (de.richsource.gradle.plugins:gwt-gradle-plugin:0.6) isn't maintained anymore and it is not compatible with GWT 2.10.0. It looks in the wrong package (com.google.gwt instead of org.gwtproject)... But as I saw you also maintain a fork of a gwt plugin (https://github.com/jiakuan/gwt-gradle-plugin) ? Maybe you can change to this plugin?

Or we do ignore the warnings for now.

I did test the plugin with our application and for us it works...

**Comment by jiakuan** [comment]
> I did update to gwt 2.10.0 however there are certain problems:
> 
> The gradle plugin  (de.richsource.gradle.plugins:gwt-gradle-plugin:0.6) isn't maintained anymore and it is not compatible with GWT 2.10.0. It looks in the wrong package (com.google.gwt instead of org.gwtproject)... But as I saw you also maintain a fork of a gwt plugin (https://github.com/jiakuan/gwt-gradle-plugin) ? Maybe you can change to this plugin?
> 
> 
> 
> Or we do ignore the warnings for now.
> 
> 
> 
> I did test the plugin with our application and for us it works...

Great, thanks!

I agree with your previous comment. We could leave the GWT upgrade and plug-in upgrade to a separate PR. I will check and merge this one first.

**Review by jiakuan — APPROVED** [review]

**Comment by Lonzak** [comment]
ah sorry - I did also commit the gwt. 2.10.0 update to this PR ...
Do you plan to create  a release out of it? That would be nice...

**Comment by jiakuan** [comment]
No problem.

I've merged and I will need to fix some issues and cleanup the project structure a little bit.

After that, I will create a new release.

**Pull request merged**